scripts/kmodtool

```diff
@@ -144,6 +144,12 @@ print_rpmtemplate_per_kmodpkg ()
 	local kernel_uname_r=${1}
 	local kernel_variant="${2:+-${2}}"
 
+	# With RHEL 8, depmod moved from /sbin to /usr/sbin
+	local depmod_path=/sbin/depmod
+	if [ ! -f ${depmod_path} ]; then
+		depmod_path=/usr/sbin/depmod
```

Comment:
It looks as is we should also be using `depmod_path` in the `%post` and `%postun` sections.